sopel/coretasks.py

```diff
@@ -162,6 +162,121 @@ def retry_join(bot, trigger):
     bot.join(channel)
 
 
+@sopel.module.rule('.*')
+@sopel.module.event(events.RPL_MYINFO)
+@sopel.module.priority('high')
+@sopel.module.thread(False)
+@sopel.module.unblockable
+def parse_reply_myinfo(bot, trigger):
+    """Parse RPL_MYINFO for servername.
+
+    004 RPL_MYINFO "<client> <servername> <version> <available user modes>
+    <available channel modes> [<channel modes with a parameter>]"
+    """
+    bot.server.hostname = trigger.args[1]
+
+
+@sopel.module.rule('.*')
+@sopel.module.event(events.RPL_ISUPPORT)
+@sopel.module.priority('high')
+@sopel.module.thread(False)
+@sopel.module.unblockable
+def parse_reply_isupport(bot, trigger):
+    """Parse RPL_ISUPPORT.
+
+    005 RPL_ISUPPORT "<client> <1-13 tokens> :are supported by this server"
+    """
+    DEFAULT_VALUES = {
```

Comment:
The number of constants just for ISUPPORT has been moved under `sopel.tools.isupport` because of how many constants (and how many lines for each constant object).